pkg/Cpanel/Security/Advisor/Assessors/EntropyChat.pm

```diff
@@ -46,7 +46,7 @@ sub _check_entropy_chat_enabled {
     if ( Cpanel::RestartSrv::check_service( 'service' => 'entropychat', 'user' => 'nobody' ) ) {
         $security_advisor_obj->add_advice(
             {
-                'type'       => $Cpanel::Security::Advisor::ADVISE_BAD,
+                'key' => 'EntropyChat_is_running', 'type' => $Cpanel::Security::Advisor::ADVISE_BAD,
```

Comment:
If perltidy permits it, I might recommend splitting this into two lines.  That's a nice-to-have, though.